tests/testthat/test-get_code.R

```diff
@@ -33,6 +33,17 @@ testthat::test_that("handles the code included in curly brackets", {
   )
 })
 
+testthat::test_that("handles the code of length > 1 included in curly brackets", {
```

Comment:
This test has some syntax issues. Also I wouldn't test warning message here but use `eval_code(teal.data::teal_data(), ...)`

```suggestion
testthat::test_that("handles the code of length > 1 when at least one is enclosed in curly brackets", {
```